src/styles/breadcrumb.styl

```diff
@@ -8,6 +8,8 @@
     color #95999d
     a
         cursor pointer
+    .separator
```

Comment:
Sort of unrelated to the PR, but the spacing in the breadcrumb was relying on literal spaces and was really inconsistent. Now it's handled with css!